wbooks/src/app/index.tsx

```diff
@@ -10,38 +10,89 @@
 import 'react-native-gesture-handler';
 import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { NavigationContainer } from '@react-navigation/native';
 import { RootStackParamList } from '@constants/types';
-import Colors from '@constants/colors';
 import { Image } from 'react-native';
+import Colors from '@constants/colors';
+import Library from '@screens/Library';
+import Settings from '@screens/Settings';
+import BookDetail from '@screens/BookDetail';
 
-import Library from './screens/Library';
-import BookDetail from './screens/BookDetail';
 import styles from './style';
 
-const Stack = createStackNavigator<RootStackParamList>();
-
 const HeaderBackground = () => (
   <Image source={require('@assets/General/bc_nav_bar.png')} resizeMode="contain" style={styles.header} />
 );
 
 const BackButton = () => <Image source={require('@assets/NavigationBar/ic_back.png')} />;
 
+const LibraryStack = createStackNavigator<RootStackParamList>();
```

Comment:
Move all the stack and tab navigator declaration to `src/components/AppNavigator`, it's a big mess this file:

```
<NavigationContainer>
  <StatusBar ... />
  <AppNavigator />
</NavigationContainer>
```

Then, in the `AppNavigator` you must declare off all this and:
```
...
other navigators
...
function AppNavigator() {
  return (
    <Tab.Navigator screenOptions={...} tabBarOptions={...}
      <Tab.Screen ... />
      <Tab.Screen ... />
    </Tab.Navigator>
  );
}
```

Reply:
There is a difference in the weight of the title font, although its style is according to the given styles. Is there another difference that I am not seeing? 

![image](https://user-images.githubusercontent.com/77977855/108646411-86c56f80-7494-11eb-9910-cc9c468f535c.png)
![image](https://user-images.githubusercontent.com/77977855/108646454-afe60000-7494-11eb-9781-469dba827d21.png)

Reply:
I created the AppNavigator file, but I didn't create the alias, because the location I don't know if it was the best src/components/AppNavigator. I have another alias that I use @components that is inside app/

Is that correct? Would you suggest me some aliases for this one?